Holodeck/Holodeck.py

```diff
@@ -34,7 +34,7 @@ def _get_worlds_map():
     return holodeck_worlds
 
 
-def make(world, gl_version=GL_VERSION.OPENGL4):
+def make(world, gl_version=GL_VERSION.OPENGL4, primary_cam_size=None):
```

Comment:
Nits: I think a name like "resolution" would be better here. What do you think?

Reply:
Also, make sure you are passing the codacy checks below, and you'll need to fix some merge conflicts with the new names of the files etc.